src/components/ProgressViewer/ProgressViewer.tsx

```diff
@@ -0,0 +1,97 @@
+import cn from 'bem-cn-lite';
+
+import {ValueOf} from '../../types/common';
+
+import './ProgressViewer.scss';
+
+const b = cn('progress-viewer');
+
+export const PROGRESS_VIEWER_SIZE_IDS = {
+    xs: 'xs',
+    s: 's',
+    ns: 'ns',
+    m: 'm',
+    n: 'n',
+    l: 'l',
+    head: 'head',
+} as const;
+
+type ProgressViewerSize = ValueOf<typeof PROGRESS_VIEWER_SIZE_IDS>;
+
+/*
+
+Описание props:
+1) value - величина прогресса
+2) capacity - предельно возможный прогресс
+3) formatValues - функция форматирования value и capacity
+4) percents - отображать ли заполненость в виде процентов
+5) className - кастомный класс
+6) colorizeProgress - менять ли цвет полосы прогресса в зависимости от его величины
+7) inverseColorize - инвертировать ли цвета при разукрашивании прогресса
+*/
```

Comment:
It should be translated to English. Some descriptions probably could be omitted (e.g. `classname`)